### DISCUSSION TITLE
Install an old version on Ubuntu

### DISCUSSION THREAD

**Discussion by paul0008** [streamlink/streamlink#5313] [General]
### Description

Subject: How to install Streamlink 5.3.1 on Ubuntu

I have downloaded  streamlink-5.3.1.tar.gz  from  https://github.com/streamlink/streamlink/releases/tag/5.3.1
Can I use this file to install streamlink 5.3.1 and how ?

Can somebody here please post how should I normally go about if I wanted to install an old version of streamlink on Ubuntu?

**Comment by bastimeyer** [comment]
Read the linked install instructions on the link you've posted.

**Comment by paul0008** [comment]
Thanks. This worked:

python3 -m pip install ./streamlink-5.3.1.tar.gz
